peoplehr/types.bal

```diff
@@ -0,0 +1,863 @@
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+ 
+type Authentication record {
+    string APIKey?;
+    string Action?;
+};
+
+# Represents common fields of the response
+#
+# + isError - Indicates whether the response is an error or not
+# + Status - The status of the error  
+# + Message - The message of the response  
+public type PeopleHrResponse record {
```

Comment:
Shall we rename to `PeopleHRGenricResponse`? Check if we can make this a private type as users will not use it directly.